src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -284,6 +305,49 @@ static bool ShouldBeFilteredOutOfExpandedCompletionList(VSCompletionItem item, I
                 // 2. or, all associated expanders are unselected, therefore should be excluded
                 return associatedWithUnselectedExpander;
             }
+
+            static ImmutableArray<CompletionItemWithHighlight> GetHighlightedList(
+                ImmutableArray<ExtendedFilterResult> filterResults,
+                string filterText,
+                CompletionHelper completionHelper,
+                bool highlightMatchingPortions)
+            {
+                if (!highlightMatchingPortions)
+                {
+                    return filterResults.SelectAsArray(r => new CompletionItemWithHighlight(r.VSCompletionItem, ImmutableArray<Span>.Empty));
+                }
```

Comment:
feels like this should be pulled into the caller (unless this is called in multiple places).

Reply:
Done